### PR TITLE
Disable button on submit in forms

### PR DESCRIPTION
**Pull request by feeds**
See #267

### CODE REVIEW

**Review by josauder — APPROVED** [review]
lgtm

File: backend/apps/iamstudent/forms.py [hunk 1]
```diff
@@ -269,8 +272,8 @@ def __init__(self, *args, **kwargs):
             'einwilligung_datenweitergabe',
             HTML("</p>"),
             HTML('<div class="registration_disclaimer">{}</div>'.format(_('Die Bereitstellung unseres Services erfolgt unentgeltlich. Mir ist bewusst, dass die Ausgestaltung des Verhältnisses zur Institution allein mich und die entsprechende Institution betrifft. Insbesondere Art und Umfang der Arbeit, eine etwaige Vergütung und vergleichbares betreffen nur mich und die entsprechende Institution. Eine Haftung von match4healthcare ist ausgeschlossen.'))),
-            Submit('submit', _('Registriere mich'), css_class='btn blue text-white btn-md',
-                   onclick="this.form.submit(); this.disabled=true; this.value='Sending…';"),
+            Submit('submit', _('Registriere mich'), css_class='btn blue text-white btn-md',id='studentFormButton'),
+            HTML("<script>function disableButton() {var btn = document.getElementById('btstudentFormButtonn'); btn.disabled = true;btn.innerText = 'Sending...'}</script>")
```
Comment (kevihiiin) [inline backend/apps/iamstudent/forms.py]: Typo in ID for the button

Comment (kevihiiin) [reply backend/apps/iamstudent/forms.py]: Crispy Forms, adds their own id (before we can add ours) and displays the button text as value.

**Review by kevihiiin — APPROVED** [review]
Nice fix!
The automatically generated submit button (on the student page) has different values and other ID than the one in the hospital page

**Pull request merged**